frontend/components/LiveQuery/SelectTargets.tsx

```diff
@@ -348,7 +348,8 @@ const SelectTargets = ({
 
     return (
       <>
-        <span>{total}</span>&nbsp;hosts targeted&nbsp; ({onlinePercentage}
+        <span>{total}</span>&nbsp;host{total > 1 ? `s` : ``} targeted&nbsp; (
```

Comment:
Love this copy fix, can we include host/hosts fix for PolicyPage's `QueryResults.tsx` and  QueryPage's `QueryResults.tsx`:

<img width="591" alt="Screen Shot 2022-09-23 at 4 07 02 PM" src="https://user-images.githubusercontent.com/71795832/192048825-75875371-9d4b-439e-aceb-eda302a92596.png">

Reply:
Ooouuu, good catch! Fixed.